mock/service/controller.go

```diff
@@ -388,8 +420,19 @@ func (s *service) ListVolumes(
 	)
 
 	for i = 0; i < len(entries); i++ {
+		volumeStatus, err := s.NodeGetVolumeStats(ctx, &csi.NodeGetVolumeStatsRequest{
```

Comment:
ListVolumes is controller PRC.  We should not call a Node PRC within a controller RPC.

Reply:
Sure. Thanks for your comment.